send-data/aws-overview.mdx

```diff
@@ -0,0 +1,44 @@
+
+The table below summarizes the methods you can use to send data from different Amazon services to Axiom.
+
+| Supported Amazon service                             | Supported methods to send data to Axiom                                                                                              |
+|------------------------------------------------------|--------------------------------------------------------------------------------------------------------------------------------------|
+| API Gateway                                          | [AWS Cloudwatch Forwarder](/send-data/cloudwatch)                                                                                    |
+| AppSync                                              | [AWS Cloudwatch Forwarder](/send-data/cloudwatch)                                                                                    |
+| Aurora MySQL                                         | [AWS Cloudwatch Forwarder](/send-data/cloudwatch)                                                                                    |
+| Beanstalk                                            | [AWS Cloudwatch Forwarder](/send-data/cloudwatch)                                                                                    |
+| Bedrock Knowledge                                    | [AWS Cloudwatch Forwarder](/send-data/cloudwatch), [AWS S3 Forwarder](/send-data/aws-s3) or [AWS Firehose](/send-data/aws-firehose)  |
+| Chime media quality metric logs and SIP message logs | [AWS Cloudwatch Forwarder](/send-data/cloudwatch)                                                                                    |
+| Cloudfront                                           | [AWS S3 Forwarder](/send-data/aws-s3)                                                                                                |
+| CloudHSM audit logs                                  | [AWS Cloudwatch Forwarder](/send-data/cloudwatch)                                                                                    |
+| Cloudtrail                                           | [AWS Cloudwatch Forwarder](/send-data/cloudwatch)                                                                                    |
+| Cloudwatch                                           | [AWS Cloudwatch Forwarder](/send-data/cloudwatch)                                                                                    |
+| CloudWatch (evidently evaluation event logs)         | [AWS Cloudwatch Forwarder](/send-data/cloudwatch) or [AWS Firehose](/send-data/aws-firehose)                                         |
```

Comment:
`evidently evaluation event logs` do we need to say this here? Can this be highlighted in the docs on Cloudwatch Forwarder?